contexts/audit/ruby/Makefile

```diff
@@ -0,0 +1,14 @@
+-include ../../prerequisites/tasks.mk
+-include ../../prerequisites/ruby.mk
+
+##>> run bundle audit
```

Comment:
I don't think the 'bundle' part is needed. 
I believe something like `run audit for ruby` makes more sense in this case.

Reply:
Right. Changed.